modules/backend/widgets/mediamanager/assets/js/mediamanager.js

```diff
@@ -837,7 +838,7 @@
     MediaManager.prototype.uploadError = function(file, message) {
         this.updateUploadBar('error', 'progress-bar progress-bar-danger');
 
-        if (file.xhr.status === 413) {
+        if (file?.xhr?.status === 413) {
```

Comment:
What does this change accomplish?

Reply:
When an error relating to the filesize by dropzone was coming up it was trying to open an alert in `$.wn.alert(message)` but this was preventing it because `file.xhr` didn't exist